go.mod

```
@@ -1,10 +1,11 @@
 module github.com/jetstack/kube-oidc-proxy
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.23.0
+toolchain go1.23.4
```

```suggestion
toolchain go1.23.11
```
why not use the latest patch that fixes more vulnerabilities